.github/workflows/codeql.yml

```
@@ -0,0 +1,89 @@
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
```

actionlint (1.7.7)

"schedule" section must be sequence node but got scalar node with "!!null" tag

(syntax-check)